app/e2e/client/rocketchat.e2e.js

```diff
@@ -145,26 +148,30 @@ class E2E extends Emitter {
 			});
 
 			this.openAlert({
-				title: TAPi18n.__('Save_your_encryption_password'),
+				title: TAPi18n.__('Save_Your_Encryption_Password'),
 				html: TAPi18n.__('Click_here_to_view_and_copy_your_password'),
 				modifiers: ['large'],
 				closable: false,
 				icon: 'key',
 				action: () => {
-					modal.open({
-						title: TAPi18n.__('Save_your_encryption_password'),
-						html: true,
-						text: `<div>${ passwordRevealText }</div>`,
-						showConfirmButton: true,
-						showCancelButton: true,
-						confirmButtonText: TAPi18n.__('I_saved_my_password_close_this_message'),
-						cancelButtonText: TAPi18n.__('I_ll_do_it_later'),
-					}, (confirm) => {
-						if (!confirm) {
-							return;
-						}
-						Meteor._localStorage.removeItem('e2e.randomPassword');
-						this.closeAlert();
+					imperativeModal.open({ component: GenericModal,
+						props: {
+							variant: 'warning',
+							title: TAPi18n.__('Save_Your_Encryption_Password'),
+							children: <div dangerouslySetInnerHTML={{ __html: passwordRevealText }} />,
```

Comment:
As we agreed previously, we should avoid JSX at all costs when dealing with legacy files. Let's keep JSX for pure React files.